src/Microsoft.OpenApi.CSharpAnnotations.DocumentGeneration/OpenApiGeneratorConfig.cs

```diff
@@ -95,5 +95,10 @@ public class OpenApiGeneratorConfig
         /// document from C# annotations.
         /// </summary>
         public OpenApiGeneratorFilterConfig OpenApiGeneratorFilterConfig { get; }
+
+        /// <summary>
+        /// The description to use while populating OpenApiInfo.
+        /// </summary>
+        public string OpenApiInfoDescription { get; set; }
```

Comment:
Add a new constructor that populates this.

Reply:
i do not think we need constructor for each optional setting

---
In reply to: [221388432](https://github.com/Microsoft/OpenAPI.NET.CSharpAnnotations/pull/158#discussion_r221388432) [](ancestors = 221388432)